File: 01_Programming_Basics/03_Programming_Basics_with_JavaScript/04_Nested_Conditional_Statements/02_SmallShop.js
function SmallShop(args) {
  let product = args[0];
  let town = args[1].toLowerCase();
  let quantity = parseFloat(args[2]);
  let price = 0;

  if (town == "sofia") {
    if (product == "coffee") {price = quantity * 0.50; }
    if (product == "water") {price = quantity * 0.80; }
    if (product == "beer") {price = quantity * 1.20; }
    if (product == "sweets") {price = quantity * 1.45; }  
    if (product == "peanuts") {price = quantity * 1.60; }         
  } else if (town == "plovdiv") {
    if (product == "coffee") {price = quantity * 0.40; }
    if (product == "water") {price = quantity * 0.70; }
    if (product == "beer") {price = quantity * 1.15; }
    if (product == "sweets") {price = quantity * 1.30; }  
    if (product == "peanuts") {price = quantity * 1.50; }         
  } else if (town == "varna") {
    if (product == "coffee") {price = quantity * 0.45; }
    if (product == "water") {price = quantity * 0.70; }
    if (product == "beer") {price = quantity * 1.10; }
    if (product == "sweets") {price = quantity * 1.35; }  
    if (product == "peanuts") {price = quantity * 1.55; }         
  }

  console.log(price);
}

File: 01_Programming_Basics/01_Programming_Basics_with_C++/06_Nested_Conditional_Statements_Lab/02_Small_Shop.cpp
#include <iostream>
using namespace std;

int main() {
  string product = "";
  string town = "";
  double quantity = 0.0;
  double price = 0.0;

  cin >> product >> town >> quantity;

  if (town == "Sofia") {
    if (product == "coffee") {
      price = 0.50;
    } else if (product == "water") {
      price = 0.80;
    } else if (product == "beer") {
      price = 1.20;      
    } else if (product == "sweets") {
      price = 1.45;
    } else if (product == "peanuts") {
      price = 1.60;
    }
  } else if (town == "Plovdiv") {
    if (product == "coffee") {
      price = 0.40;
    } else if (product == "water") {
      price = 0.70;
    } else if (product == "beer") {
      price = 1.15;      
    } else if (product == "sweets") {
      price = 1.30;
    } else if (product == "peanuts") {
      price = 1.50;
    }
  } else if (town == "Varna") {
    if (product == "coffee") {
      price = 0.45;
    } else if (product == "water") {
      price = 0.70;
    } else if (product == "beer") {
      price = 1.10;      
    } else if (product == "sweets") {
      price = 1.35;
    } else if (product == "peanuts") {
      price = 1.55;
    }
  }

  cout << price * quantity << endl;

  return 0;

}